backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -138,6 +138,54 @@ module DarkDateTime =
   let toIsoString (d : T) : string = (toInstant d).toIsoString ()
 
 
+/// Dark runtime type
+type DType =
```

Comment:
Note: in a few files, I had to move the 'dtype' module/type def above the Expr module/type def, because EApply now references DType